ansible/roles/dashd/templates/dash.conf.j2

```diff
@@ -78,10 +78,13 @@ sporkkey={{ dashd_sporkkey }}
 [regtest]
 {% elif dash_network == 'mainnet' %}
 [main]
+llmq-qvvec-sync=llmq_100_67:0
 {% elif dash_network == 'testnet' %}
 [test]
+llmq-qvvec-sync=llmq_100_67:0
 {% elif dash_network == 'devnet' %}
 [devnet]
+llmq-qvvec-sync=llmq_50_60:0
```

Comment:
Actually, this option must be aligned with `platform_drive_validator_set_llmq_type`. We can remove it from condition and set the value according to `platform_drive_validator_set_llmq_type`. You can create a map in `group_vars/all` - `core_llmq_types` like:
```
core_llmq_types:
    1: llmq_50_60
    2: llmq_400_60
    3: llmq_400_85
    4: llmq_100_67
    100: llmq_test
    101: llmq_devnet
    102: llmq_test_v17
```

and then set the option in config:

```
core_llmq_types[platform_drive_validator_set_llmq_type]
```